### PR TITLE
DOCS 2942 Update SCALE System Settings Support

### PR DESCRIPTION
**Pull request by jervin777**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALEUIReference/SystemSettings/GeneralSettings.md [hunk 1]
```diff
@@ -42,9 +42,67 @@ If you do not save the system configuration before resetting it, you may lose da
...
 ## Support
 
-The *Support* window in the Advanced Settings screen displays the systems general hardware and software specs and contains links to the Documentation Hub, TrueNAS Forums, and enterprise licensing information. 
+The *Support* window displays the systems general hardware and software specs and contains links to the [Documentation Hub](https://www.truenas.com/docs/), [TrueNAS Forums](https://www.truenas.com/community/), and offers [TrueNAS Licensing](https://www.ixsystems.com/support/) information. 
 
-There are also buttons that allow users to add an enterprise license or report bugs via a Jira support ticket.
+![GeneralSettingsSCALESupport](/images/SCALE/GeneralSettingsSCALESupport.png "SCALE General Settings Support View")
+
+{{< tabs "Support Options" >}}
+{{< tab "Add License" >}}
```
Comment (Mrt134) [inline content/SCALE/SCALEUIReference/SystemSettings/GeneralSettings.md:50]: @jervin777 I'm going to rework the article to no longer use tabs. Our current theme doesn't support linking inside tabbed sections and it's causing reader confusion.

Comment (Mrt134) [inline content/SCALE/SCALEUIReference/SystemSettings/GeneralSettings.md:50]: @jervin777 Can you rework this article to no longer use tabs? Instead, lets go for a header and expand option as this allows readers to link to specific areas but the vertical scroll of the article is still reduced.
```suggestion
### Add License
{{< expand "Click to expand" "v" >}}
.....<content>
{{< /expand >}}
```

**Review by Mrt134 — CHANGES_REQUESTED** [review]
Requesting rework with tabs removed and replaced with headers and expands.

File: content/SCALE/SCALEUIReference/SystemSettings/GeneralSettings.md (hunk 1 above)
Comment (jervin777) [reply content/SCALE/SCALEUIReference/SystemSettings/GeneralSettings.md:50]: Yes

**Review by Mrt134 — APPROVED** [review]

**Comment by Mrt134** [comment]
Review looks good - link check failed but this PR unaffected by that error.

**Pull request merged**